features/cli_help.feature

```diff
@@ -11,7 +11,7 @@ Feature: Howitzer CLI Help
         howitzer [global options] command [command options] [arguments...]
 
     VERSION
-        <HOWITZER_VERSION>
+        2.0.3
```

Comment:
The test will be broken next release. Please convert transformer to new format in Cucumber acceptance tests also